SUMMARY.md

```diff
@@ -117,6 +117,12 @@
     - [Registers](./hw/ip/aon_timer/doc/registers.md)
     - [Device Interface Functions](./sw/device/lib/dif/dif_aon_timer.h)
     - [Checklist](./hw/ip/aon_timer/doc/checklist.md)
+  - [ASCON](./hw/ip/ascon/README.md)
```

Comment:
```suggestion
  - [Ascon](./hw/ip/ascon/README.md)
```

From what I've seen in the docs in this PR, the common capitalization is 'Ascon', so let's use that here too, please